### PR TITLE
12 check animation on hover item in menu

### LINKED ISSUE #12: Check_animation_on_hover_item_in_menu

### PR DESCRIPTION
**Pull request by Marcelin97**
close #12

### CODE REVIEW

File: assets/scss/home.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@
   justify-content: center;
   align-items: center;
   display: flex;
-  animation: hidden 4s ease-in-out forwards;
+  animation: hidden 0s ease-in-out forwards;
```
Comment (Tutt-J) [inline assets/scss/home.scss]: Why 0 ?

File: assets/scss/menu.scss [hunk 2]
```diff
@@ -109,39 +111,87 @@ article {
...
+      border-radius: 0 15px 15px 0;
+      position: relative;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      width: 0px;
+      left: 20px;
+      height: 80px;
+      transition-property: width;
+      transition-duration: 0.7s;
```
Comment (Tutt-J) [inline assets/scss/menu.scss]: combine property

File: assets/scss/menu.scss [hunk 3]
```diff
@@ -109,39 +111,87 @@ article {
...
+          color: white;
+          font-size: 1.5em;
+        }
+    }
+    &:hover div:nth-child(3) {/*check au survol*/
+      display: flex;
+      left: 15px;
+      display: flex;
+      width: 80px;
+      transition-property: width;
```
Comment (Tutt-J) [inline assets/scss/menu.scss]: why not height 100% ?

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: assets/scss/home.scss (hunk 1 above)
Comment (Marcelin97) [reply assets/scss/home.scss]: to work without constant loading.

File: assets/scss/menu.scss (hunk 3 above)
Comment (Marcelin97) [reply assets/scss/menu.scss]: add height

File: assets/scss/menu.scss (hunk 2 above)
Comment (Marcelin97) [reply assets/scss/menu.scss]: i use transition property.

**Review by Tutt-J — APPROVED** [review]

**Pull request merged**